test/scripts/bin/proxysql-tester.py

```
@@ -0,0 +1,1777 @@
+        if display:
+            log.debug(result)
+        return result
+
+    def log_test_run(self, run):
+        log_path = f'{tests_logs_path}/{script}'
+        os.makedirs(log_path, exist_ok=True)
+        #log.addHandler(logging.FileHandler(f'{log_path}/{run}.log', 'w'))
+        log.info(f"Running '{run}' ...")
+
+        # run one of the tests 'run_XXXXX'
+        rc, result, summary = getattr(psqlt, f'run_{run}')
```

Ruff (0.15.6)

[error] Undefined name `psqlt`

(F821)